### DISCUSSION TITLE
Altervista

### DISCUSSION THREAD

**Discussion by scuolagioco** [iisgiua/giuaschool#175] [Richieste di aiuto]
Buonasera, siamo ancora qui a fare richiesta di aiuto per l'installazione con Altervista😅… abbiamo contattato privatamente il sig. @manuelbarberi, senza avere risposta. Mi dica lei sig Barberi quando è disponibile perché avremmo davvero bisogno del registro.

**Comment by manuelbarberi** [comment]
In questo periodo sono stato molto occupato; mi rimandi tutti i dettagli e provvederò all'installazione
